shared/process/exec.rb

```diff
@@ -12,9 +12,8 @@
   end
 
   it "raises Errno::EACCES when the file does not have execute permissions" do
-    File.executable?(__FILE__).should == false
     lambda { @object.exec __FILE__ }.should raise_error(Errno::EACCES)
-  end
+  end unless File.executable?(__FILE__) # Some FS (e.g. vboxfs) locate all files executable
```

Comment:
I think it might be clearer to wrap the spec like this:
```ruby
  unless File.executable?(__FILE__)
    it "raises Errno::EACCES when the file does not have execute permissions" do
    ...
  end
```
to match the other guards style.